src/main/scala/com/microsoft/partnercatalyst/fortis/spark/analyzer/ExtendedFortisEvent.scala

```diff
@@ -0,0 +1,21 @@
+package com.microsoft.partnercatalyst.fortis.spark.analyzer
+
+import java.util.UUID
+
+import com.microsoft.partnercatalyst.fortis.spark.dto.{Analysis, Details, FortisEvent, Location}
+
+case class ExtendedFortisEvent[T](
+  details: ExtendedDetails[T],
+  analysis: Analysis
+) extends FortisEvent
+
+case class ExtendedDetails[T](
+  id: UUID,
+  createdAtEpoch: Long,
+  body: String,
+  title: String,
+  publisher: String,
+  sourceUrl: String,
+  sharedLocations: List[Location] = List(),
+  original: T
```

Comment:
Why do we need `original: T` here? As per [previous discussion](https://github.com/CatalystCode/project-fortis-spark/pull/22#issuecomment-310201771), we don't need to keep around the original item. Not keeping the item around (a) ensures that we don't encounter any surprises in the last stage of the pipeline when we try to persist all the disparate schemas to Cassandra and (b) minimizes the amount of data we have to unnecessarily serialize for every stage of the Spark pipeline.